Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Stage 1: Build
+
+FROM node:20-alpine AS build
+
+RUN npm install -g pnpm
+
+WORKDIR /app
+COPY . .
+
+# ? Consider --frozen-lockfile
+RUN pnpm install
+
+# ? Add checks?
+
+# ? Consider --prod
+RUN pnpm run build
```

Comment:
afaik, we can make production artifact by simply using `pnpm run build` so we don't need to add --prod flag